tests/tests_pytorch/profilers/test_profiler.py

```diff
@@ -471,20 +471,21 @@ def look_for_trace(trace_dir):
     assert look_for_trace(tmpdir / "lightning_logs" / "version_0")
 
 
-@RunIf(min_cuda_gpus=1, standalone=True)
-def test_pytorch_profiler_nested_emit_nvtx():
-    """This test check emit_nvtx is correctly supported."""
-    profiler = PyTorchProfiler(use_cuda=True, emit_nvtx=True)
-    model = BoringModel()
-    trainer = Trainer(
-        fast_dev_run=True,
-        profiler=profiler,
-        accelerator="gpu",
-        devices=1,
-        enable_progress_bar=False,
-        enable_model_summary=False,
-    )
-    trainer.fit(model)
+# @RunIf(min_cuda_gpus=1, standalone=True)
```

Comment:
Maybe add a TODO or some comment explaining:
- why it is commented out
- why it isn't just deleted

Reply:
```suggestion
# Todo: this test has not been running as all our CI GPU runners have higher capacity
# @RunIf(min_cuda_gpus=1, standalone=True)
```